docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+- Add Link (link to relevant resources such as the publication itself, lab website, consortia/department website, entry in GEO or other raw data sources, etc.)
+
+You will also be required to agree to CZI's data submission policies which you can read in full by clicking 'Show Details'
+
+### Add a dataset to a private collection
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113529554-b5556080-9591-11eb-8e0e-274fcb1ff39a.png) --->
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/25663501/113529554-b5556080-9591-11eb-8e0e-274fcb1ff39a.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> Add dataset to a collection
+</p>
+
+<br/>
+
+Once you have created a collection, you will have the ability to add a dataset. Once you click this button, you will be guided to dropbox, where you can select a dataset for upload.
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113529612-db7b0080-9591-11eb-9720-ae47155d62f8.png) --->
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/25663501/113529612-db7b0080-9591-11eb-9720-ae47155d62f8.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> Choose dataset from dropbox
+</p>
+
+<br/>
+
+
+### Remove dataset from a private collection
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113529632-edf53a00-9591-11eb-8ea4-575ca8c643f7.png) --->
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/25663501/113529632-edf53a00-9591-11eb-8ea4-575ca8c643f7.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> Remove dataset from a collection
+</p>
+
+<br/>
+
+In case you have uploaded the wrong dataset or want to update a dataset within a collection, you can delete a dataset associated with a collection (without deleting the collection itself).
+
+### Dataset actions
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/25663501/113614805-f8aae000-9620-11eb-8421-359e7ec69f86.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> Successful dataset upload
+</p>
+
+Once you have successfully uploaded a dataset to your collection, you are given options to delete the dataset, download (in `anndata` (`.h5ad`), `seurat V3` (`.rds`), or `.loom`), and explore the dataset using the cellxgene explorer. These options are given by the three icons on the right hand side of the entry.
+
+### Publish Collection to the portal
+
+<!--- ![image](https://user-images.githubusercontent.com/25663501/113615336-b930c380-9621-11eb-88ae-7855a49c5c61.png) --->
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/25663501/113615336-b930c380-9621-11eb-88ae-7855a49c5c61.png">
+</p>
+
+<p align="center">
+  <b> Figure:</b> Publish your collection
+</p>
+
+Finally, if your collection is complete and you wish to share with the world, you simply need to hit publish (in the upper right hand side of the collection view). Congratulations! :partying_face:
```

Comment:
I _hit_ publish, but it knocked my laptop off the table.

Reply:
lol